### ISSUE TITLE
Exceptions logged with Logback + SLF4J don't show up in Stackdriver

### ISSUE THREAD

**Issue by bigunyak** [GoogleCloudPlatform/google-cloud-java#2735]
I'm using the Logback + SLF4J logging configuration described in the official documentation [here](https://cloud.google.com/logging/docs/setup/java).
Everything works fine with simple log messages like:
```java
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
...
private static final Logger logger = LoggerFactory.getLogger(MyClass.class);
...
logger.info("Hello world");
```

The problem appears when I try to log exceptions like this:
```java
logger.error("Something went wrong:", ex);
```

I see only my description message in Stackdriver but not the stack trace from the provided exception.
What can be the problem here?

**Comment by int08h** [comment]
I ran into this too. Stacktraces do not get logged, nor does Stackdriver Error Reporting receive them. 

Docs say otherwise: from [Setting Up Stackdriver Error Reporting for Java](https://cloud.google.com/error-reporting/docs/setup/java)
```text
Exceptions logged using the Stackdriver Logback Appender or java.util.logging 
Stackdriver Handler are automatically reported to the Error Reporting Console.
```

May be a dupe of #2587.

**Comment by pongad** [comment]
Sorry for the delay, I just got cycles to investigate this.

Investigation so far: I was able to do this
```java
    Logger logger = Logger.getLogger("thing");
    logger.setUseParentHandlers(false);

    Handler handler = new LoggingHandler();
    logger.addHandler(handler);

    logger.log(Level.INFO,"logging thing", new IllegalStateException("foo"));
```
and the stack seems to be logged OK.

This problem is probably with the logback appender, not with logging proper. I'll continute investigating this, but also assigning to @jabubake who has more context.

**Comment by bigunyak** [comment]
Hm, I've just tried to install `google-cloud-logging-logback-0.33.0-alpha` but see no difference...
```java
logger.error("Error message", throwable);
```
Still sends only `Error message` to Stackdriver with no trace included.
Or am I doing something wrong?

**Comment by garrettjonesgoogle** [comment]
@bigunyak , the fix has been merged to master, but it has not been included in a release yet - it was merged after 0.33.0 was released.
